src/display/general/overallGraphs.go

```diff
@@ -253,8 +293,25 @@ func RenderCPUinfo(ctx context.Context,
 			case "<Resize>":
 				updateUI()
 
-			case "s": // s to stop
-				pause()
```

Comment:
Why was this case removed?

Reply:
It's not removed. I just moved it to be handled only when help menu is not being shown i.e, on the main page only